app/views/categories/show.html.erb

```diff
@@ -7,11 +7,40 @@
   <%= render 'shared/breadcrumbs', breadcrumbs: @breadcrumbs unless @breadcrumbs.empty? %>
 <% end %>
 
+<%= render 'shared/svg/icons/plus' %>
+<%= render 'shared/svg/icons/minus' %>
+
 <div class="l-main editorial">
   <div class="l-main__row">
-    <main class="l-main__cell-single" role="main">
-      <%= heading_tag category.title %>
-      <p class="intro"><%= category.description %></p>
+    <% if category.images? %>
+      <style>
+        .l-category--image {
+          background-image: url(<%= category.images['small'] %>);
+        }
+
+        @media only screen and (-webkit-min-device-pixel-ratio: 2),
+        only screen and (min--moz-device-pixel-ratio: 2),
+        only screen and (-o-min-device-pixel-ratio: #{(2*10)}/10),
+        only screen and (min-device-pixel-ratio: 2),
+        only screen and (min-resolution: #{(2*96)}dpi),
+        only screen and (min-resolution: #{2}dppx) {
+          .l-category--image {
+            background-image: url(<%= category.images['large'] %>);
+          }
+        }
+      </style>
```

Comment:
Is there a better place for this? And the SVGs? There's a lot of set-up and style going on here